content/docs/SDKs/python/agent-to-agent.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Agent-to-Agent Communication
+description: Enable communication between agents in the Agentuity Python SDK
+---
+
+# Agent-to-Agent Communication
+
+The Agentuity Python SDK enables seamless communication between agents, whether they are running locally or in different environments.
+
+## Local Agent Communication
+
+You can easily communicate between agents running in the same process:
+
+```python
+async def run(request, response, context):
+    # Call another agent by ID
+    return await response.handoff({"id": "other_agent_id"})
+    
+    # Or call by name
+    # return response.handoff({"name": "OtherAgentName"})
+```
+
+## Remote Agent Communication
+
+The SDK now supports communication with agents running in different environments:
+
+```python
+async def run(request, response, context):
+    # Resolve an agent (local or remote)
+    agent = context.get_agent("remote_agent_id")
+    
+    # Create data for the remote agent
+    from agentuity.server.data import Data, StringStreamReader
+    data = Data("text/plain", StringStreamReader("Hello from another agent"))
+    
+    # Call the remote agent
+    result = await agent.run(data, metadata={"key": "value"})
+    
+    # Process the response
+    text = await result.data.text()
+    return response.text(f"Got response: {text}")
+```
+
+## Agent Resolution
+
+The `resolve_agent` function automatically determines whether to create a local or remote agent instance:
+
+```python
+from agentuity.server.agent import resolve_agent
+
+async def run(request, response, context):
+    # Resolve by ID
+    agent = resolve_agent(context, "agent_id")
```

Comment:
use `context.get_agent({"id": "agent_id"})` instead of resolve_agent which is meant to be internal